roles/maas_postgres/tasks/o11y_pg.yaml

```diff
@@ -0,0 +1,60 @@
+---
+- name: "Create postgres-exporter directory"
+  ansible.builtin.file:
+    path: "{{ o11y_postgres_exporter_dir }}"
+    state: "{{ 'directory' if (o11y_enable and 'maas_postgres' in group_names) else 'absent' }}"
+    mode: 0755
+  changed_when: false
+
+- name: "Discover host architecture"
+  ansible.builtin.command: dpkg --print-architecture
+  register: ar_out
+  changed_when: false
+
+- name: "Set arch facts"
+  ansible.builtin.set_fact:
+    arch: "{{ ar_out.stdout }}"
+    pg_exp_pkg: "https://github.com/prometheus-community/postgres_exporter/releases/download/v0.11.1/postgres_exporter-0.11.1.linux-{{ ar_out.stdout }}.tar.gz"
```

Comment:
move this URL to group_vars/all, it wll be easier to upgrade packages later if we keep everything in the same place